spark/common/src/test/scala/org/apache/sedona/sql/rasteralgebraTest.scala

```diff
@@ -931,6 +931,25 @@ class rasteralgebraTest extends TestBaseScala with BeforeAndAfter with GivenWhen
       assertEquals(expectedY, actualCoordinates.y, 1e-5)
     }
 
+    it("Passed RS_PixelAsPoints with empty raster") {
```

Comment:
Please add a test with Spark explode function and make sure the output of this function can work seamlessly with explode.